x-pack/plugins/ml/server/models/data_visualizer/data_visualizer.ts

```diff
@@ -602,7 +615,8 @@ export class DataVisualizer {
     timeFieldName: string,
     earliestMs?: number,
     latestMs?: number,
-    datafeedConfig?: Datafeed
+    datafeedConfig?: Datafeed,
+    runtimeFields?: RuntimeMappings
```

Comment:
nit, should be called `runtimeMappings` for consistency

Reply:
Updated here [`a6ae3ce` (#95252)](https://github.com/elastic/kibana/pull/95252/commits/a6ae3cef9ab00c282f56b1b517fe0f28c30e4d08)